.github/workflows/create_llvm.yml

```diff
@@ -15,18 +18,33 @@ jobs:
         version: [17, 18]
         os: [ubuntu-22.04]
         build_type: [Release, RelAssert]
+        arch : [x86_64, aarch64]
+        exclude:
+          # For now just do latest llvm with aarch to reduce cache usage.
+          - arch: aarch64
+            version: 17
+          - arch: aarch64
+            build_type: Release
         include:
           # We want to set flags related to particular matrix dimensions. To do this
           # we need to create default values first, and then against particular matrix
           # dimensions.
           # Note that we need to use RelAssert as the cache key matching can match Release against ReleaseAssert
           - os_flags:
+          - arch_flags:
           - build_type_flags:
-          - build_type: RelAssert
-            build_type_flags: -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON
+          - build_type_flags: -DCMAKE_BUILD_TYPE=Release -DLLVM_ENABLE_ASSERTIONS=ON
+            build_type: RelAssert
+          - arch_flags: -DCMAKE_TOOLCHAIN_FILE="$GITHUB_WORKSPACE/ock/platform/arm-linux/aarch64-toolchain.cmake"
+                        -DLLVM_TARGET_ARCH=AArch64
+                        -DLLVM_HOST_TRIPLE=aarch64-unknown-linux-gnu
+                        -DLLVM_DEFAULT_TARGET_TRIPLE=aarch64-unknown-linux-gnu
```

Comment:
`LLVM_DEFAULT_TARGET_TRIPLE` should not be necessary, it defaults to `LLVM_HOST_TRIPLE`.

Reply:
okay, but again it's in the documentation